demonstrations/ahs_aquila.py

```diff
@@ -0,0 +1,812 @@
+
+.. math:: \hat{H}_{int} = \sum_{j=1}^{N-1}\sum_{k=j+1}^{N} V_{jk}\hat{n}_j\hat{n}_k = \sum_{j=1}^{N-1}\sum_{k=j+1}^{N} \frac{C_6}{R^6_{jk}}\hat{n}_j\hat{n}_k
+
+where :math:`n_j=\ket{r_j}\bra{r_j}` is the number operator acting on atom :math:`j`, :math:`R_{jk} = \lvert x_j - x_k \lvert` is the
+distance between atoms :math:`j` and :math:`k`, and :math:`C_6` is a fixed value determined by the nature of the ground
+and Rydberg states (for Aquila, :math:`5.24 \times 10^{-24} \text{rad m}^6 / \text{s}`, referring to the
+:math:`| 70S_{1/2} \rangle` state of the Rb-87 atom).
+
+There are two key things to be aware of in the interaction term. First, the energy contribution of
+the interaction between each pair of atoms is only non-zero when both atoms are in the Rydberg state,
+so that :math:`\bra{\psi}\hat{n}_k \hat{n}_j\ket{\psi}=1`. Second, the energy contribution for each
+pair of atoms is inversely proportional to the distance between them. Thus, as we move two atoms closer
+together, it becomes increasingly energetically expensive for both to be in the Rydberg state.
+
+In the Aquila system, to modify these interactions, we define a *register* (the layout of the atoms).
+
+Below is a conceptual diagram demonstrating this interaction for a pair of atoms. At a distance, a similar
+energetic cost is paid to from 0 to 1 excitation and from 1 to 2 excitations. However, as we move the
+atoms into closer proximity, we see a rapidly increasing energy cost to drive to the doubly excited state.
+
+.. figure:: ../demonstrations/ahs_aquila/rydberg_blockade_diagram.png
+    :align: center
+    :width: 50%
+    :alt: A diagram of the energy levels for the ground, single excitation, and double excitation states
+    :target: javascript:void(0);
+
+The modification of the energy levels when atoms are in proximity gives rise to Rydberg blockade,
+where atoms that have been driven by a pulse that would, in isolation, leave them in the excited state
+instead remain in the ground state due to neighboring atoms being excited.
+
+This brings us to our discussion of the second part of the Hamiltonian: the drive term.
+
+
+The driven Rydberg Hamiltonian
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The atoms in a Rydberg system can be driven by application of a laser pulse, which can be described by 3 parameters:
+amplitude (also called Rabi frequency) :math:`\Omega`, detuning :math:`\Delta`, and phase :math:`\phi`. While in
+theory, a drive pulse can be applied to individual atoms, the current control setup for the Aquila hardware only
+allows the application of a global drive pulse.
+
+Let’s look at how this plays out in the Hamiltonian describing a global drive targeting the ground
+to Rydberg state transition. The driven Hamiltonian of the system is:
+
+.. math::  \hat{H}_{drive} = \sum_{k=1}^N \frac{\Omega(t)}{2} (e^{i \phi(t)}\ket{g_k}\bra{r_k} - e^{-i \phi(t)} \ket{r_k}\bra{g_k}) - \Delta(t) \hat{n}_k
```

Comment:
```suggestion
.. math::  \hat{H}_{drive} = \sum_{k=1}^N \frac{\Omega(t)}{2} (e^{i \phi(t)}\ket{g_k}\bra{r_k} + e^{-i \phi(t)} \ket{r_k}\bra{g_k}) - \Delta(t) \hat{n}_k
```
typo :) there is a minus when writing it as Paulis $\cos(\phi) X - \sin(\phi) Y$, see also https://queracomputing.github.io/Bloqade.jl/dev/hamiltonians/

Reply:
updated